cisco_aci/datadog_checks/cisco_aci/fabric.py

```diff
@@ -209,3 +243,71 @@ def get_fabric_type(self, obj_type):
             return 'pod'
         if obj_type == 'l1PhysIf':
             return 'port'
+
+    def batch_payloads(self, devices, interfaces, collect_ts):
+        for device in devices:
+            yield NetworkDevicesMetadata(namespace=self.namespace, devices=[device], collect_timestamp=collect_ts)
+
+        payloads = []
+        for interface in interfaces:
+            if len(payloads) == PAYLOAD_METADATA_BATCH_SIZE:
+                yield NetworkDevicesMetadata(
+                    namespace=self.namespace, interfaces=payloads, collect_timestamp=collect_ts
+                )
+                payloads = []
+            payloads.append(interface)
+        if payloads:
+            yield NetworkDevicesMetadata(namespace=self.namespace, interfaces=payloads, collect_timestamp=collect_ts)
+
+    def submit_node_metadata(self, node_attrs, tags):
+        node = Node(attributes=node_attrs)
+        id_tags = ['namespace:{}'.format(self.namespace), 'system_ip:{}'.format(node.attributes.address)]
+        device_tags = [
+            'device_vendor:{}'.format(VENDOR_CISCO),
+            'device_namespace:{}'.format(self.namespace),
+            'device_hostname:{}'.format(node.attributes.dn),
+            'hostname:{}'.format(node.attributes.dn),
+            'system_ip:{}'.format(node.attributes.address),
+            'device_ip:{}'.format(node.attributes.address),
+            'device_id:{}:{}'.format(self.namespace, node.attributes.address),
+            "source:cisco-aci",
+        ]
+        device = DeviceMetadata(
+            id='{}:{}'.format(self.namespace, node.attributes.address),
+            id_tags=id_tags,
+            tags=device_tags + tags,
+            name=node.attributes.dn,
+            ip_address=node.attributes.address,
+            model=node.attributes.model,
+            fabric_st=node.attributes.fabric_st,
+            vendor=VENDOR_CISCO,
+            version=node.attributes.version,
+            serial_number=node.attributes.serial,
+            device_type=node.attributes.device_type,
+        )
+        return device.model_dump(exclude_none=True)
+
+    def create_interface_metadata(self, phys_if, address, tags, hostname):
+        eth = PhysIf(**phys_if.get('l1PhysIf', {}))
+        interface = InterfaceMetadata(
+            device_id='{}:{}'.format(self.namespace, address),
+            id_tags=tags,
```

Comment:
Should we use the interface name/id here?
For SNMP and Cisco SD-WAN we only use `interface:name` as id tags here

Reply:
ah ok, the id_tags for interfaces should just be `interface:name` then? i'll amend so it doesn't pass all the tags !

Reply:
Yeah I think that's what we usually do
The frontend queries metrics by filtering on the device id tags and grouping on the keys of interface id tags, so using only `interface` is enough